src/utils/Api.js

```diff
@@ -1,3 +1,6 @@
+require('dotenv').config({path: '../../.env'})
```

Comment:
I don't think you need this code. Create React App will itself do the work of requiring the right files for you. No need for 'dotenv' either. Try it out.

Reply:
Otherwise looks good.